config/locales/el.yml

```diff
@@ -0,0 +1,341 @@
+---
+el:
```

Comment:
It looks like nothing change in the file...

The `en` changed to `el`, which is correct ... but it looks like nothing was translated?